crates/network/src/any/mod.rs

```diff
@@ -1,14 +1,30 @@
 use crate::Network;
-use alloy_consensus::TxType;
-use alloy_eips::eip2718::Eip2718Error;
-use alloy_rpc_types_eth::{AnyTransactionReceipt, Transaction, TransactionRequest};
-use alloy_serde::WithOtherFields;
+use alloy_consensus::{TxEnvelope, TxType, TypedTransaction};
+use alloy_eips::{
+    eip2718::{Decodable2718, Eip2718Error, Encodable2718},
+    eip7702::SignedAuthorization,
+};
+use alloy_primitives::{Bytes, TxKind, B256, U256};
+use alloy_rpc_types_eth::{
+    AccessList, AnyTransactionReceipt, Block, Transaction, TransactionRequest,
+};
+use alloy_serde::{OtherFields, WithOtherFields};
 use core::fmt;
+use std::sync::OnceLock;
 
 mod builder;
 
+pub use alloy_consensus::{AnyHeader, AnyReceiptEnvelope};
+
+/// A catch-all header type for handling headers on multiple networks.
+pub type AnyRpcHeader = alloy_rpc_types_eth::Header<alloy_consensus::AnyHeader>;
+
+/// A catch-all block type for handling blocks on multiple networks.
+pub type AnyRpcBlock =
+    WithOtherFields<Block<WithOtherFields<Transaction<AnyTxEnvelope>>, AnyRpcHeader>>;
+
```

Comment:
we already have these here https://github.com/alloy-rs/alloy/blob/273d7849790106bb17f134a56d9725e6e3944f50/crates/rpc-types-eth/src/lib.rs#L26

Reply:
https://github.com/alloy-rs/alloy/pull/1460#issuecomment-2449771075

Those can't be in the `rpc-types-eth` crate as they need to import `AnyTxEnvelope` now. In addition, it's somewhat unclean to put `AnyRpc____` objects into `rpc-types-eth`. Do you thinkwe should add `rpc-types-any` as a crate?

Reply:
AnyRecipt and AnyHeader are already in alloy-consensus

can't AnyTxEnvelope be there too?

Reply:
i don't think it was a good idea to put them there in the first place. `alloy-consensus` has always been specifically `eth` consensus. It's also much more invasive to move a large type downwards instead of moving 2 aliases upwards

Reply:
followup work:
https://github.com/alloy-rs/alloy/issues/1598